.github/ISSUE_TEMPLATE/testing-checklist.md

```diff
@@ -12,8 +12,27 @@ assignees: ''
 
 *Use/modify this checklist as a more comprehensive list of core services to validate before cutting a major release.*
 
+## Test core services
+
+If the integration tests for all of the cloud providers are successful, that is a good sign!
+However, the following core services still need to be manually validated (until we can automate them).
+
+At minimum, the following services will need to be tested:
+- [ ] Log into keycloak as root user and add a user
```

Comment:
I like how the other list has links to every step - we should add links to docs or services here too (as in for the following steps)